libs/api-interfaces/src/lib/command-info.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface ICommandInfo {
+    // TODO Extend CommandMetadataBase from commands lib
```

Comment:
Give this a try now.

Reply:
Actually, it is only used on the front-end, and will be removed after the next step. This is because we will send command forms and not the schema directly to the front-end.